<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="main_menu">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="log_label">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>80</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="menu_layout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>60</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="check_data_button">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Просмотреть список</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="load_button">
          <property name="text">
           <string>Загрузить список из файла</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="add_obj_button">
          <property name="text">
           <string>Добавить объект в список</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sort_button">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Сортировать список</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="save_button">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Сохранить результат в файл</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>80</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
